templates/experiments/single_experiment_home.html

```diff
@@ -248,6 +248,7 @@ <h3 class="font-bold text-lg">Parent Experiments:
                 {% include "experiments/components/exports.html" %}
               {% endif %}
               {% include "experiments/filters.html"%}
+              {% include "experiments/date_range_filters.html" %}
```

Comment:
Is there a specific reason why you didn't include the date_range_filter's template and JS stuff in the filters.html file? Just curious. It would be nice if whatever we do here, could easily be ported as a single entity to other areas